hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcServer.java

```diff
@@ -726,9 +726,10 @@ static String getMethodName() {
    * @return List of name spaces in the federation on
    * removing the already invoked namespaceinfo.
    */
-  private Set<FederationNamespaceInfo> getNameSpaceInfo(String nsId) {
+  private Set<FederationNamespaceInfo> getNameSpaceInfo(
```

Comment:
This could be a static method.

Reply:
Thanks @goiri for your comment, I noticed that this method is currently only called in one place. So do we still need to change it to static? Could you please give me some advice. Thank you very much.

Reply:
I don't think this is too important to be honest.
There might be some minor optimization by being static but I do not think it is that important.
The main reason to make it static would be to make clear that this method is not doing anything with the attributes of the object.

Reply:
BTW, if we are changing the signature of the method, I would also add final to the arguments.

Reply:
Thank you very much for your detailed reply. I have made some modifications, please have a look again.